frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTabStatus.tsx

```diff
@@ -38,10 +45,9 @@ const LogsTabStatus: React.FC<LogsTabStatusProps> = ({ error, loaded, refresh, o
         latest {LOG_TAIL_LINES} lines. Exceptionally long lines are abridged. To view the full log
         for this task, you can{' '}
         <Button
-          isDisabled={!onDownload}
+          isDisabled={!onDownload || isLogsAvailable}
```

Comment:
Here the same, developers will feel confused why the button is disabled when `isLogsAvailable`. You should change it to `!isLogsAvailable` after you change the prop passed into this component.